packages/cli/src/commands/deploy/index.ts

```diff
@@ -868,8 +891,8 @@ const printDeploymentStatus = async (
     );
   } else {
     // print preview/production url
-    let previewUrl: string;
-    if (Array.isArray(aliasList) && aliasList.length > 0) {
+    let previewUrl: string = deploymentUrl;
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** revert?‏

```suggestion
    let previewUrl: string;
```

The `deploymentUrl value is always overwritten before `previewUrl` is read, isn't it?